Meadowlark-js/tests/e2e/endpoints/educationContents.test.ts

```diff
@@ -18,6 +18,33 @@ describe('Education contents', () => {
     contentClassDescriptor = await getDescriptorByLocation(contentClassDescriptorLocation);
   });
 
+  // The Data Standard defines EducationContent.LearningResource as a required "Choice".
+  // The MetaEd documentation notes that this "requirement" will not be enforced by the API, and in fact the ODS/API does not enforce it.
+  describe('when creating an EducationContent without shortDescription, contentClassDescriptor and learningResourceMetadataURI', () => {
+    beforeAll(async () => {
+      const contentIdentifier = generateRandomId();
+      educationContentLocation = await createResource({
+        endpoint: 'educationContents',
+        body: {
+          contentIdentifier,
+          namespace: '43211',
+        },
+      });
+    });
+
+    it('should create the education content', async () => {
+      await baseURLRequest()
```

Comment:
Not sure this check is necessary? If it is, can you check there is one result in the array?